### PR TITLE
CAD-425:  journald logging

### PR DESCRIPTION
**Pull request by deepfire**
Task: https://jira.iohk.io/browse/CAD-425

### CODE REVIEW

File: cardano-config/src/Cardano/Config/Logging.hs [hunk 1]
```diff
@@ -227,6 +231,11 @@ loggingCardanoFeatureInit disabled' conf = do
   Cardano.BM.Backend.Monitoring.plugin logConfig trace switchBoard
       >>= loadPlugin switchBoard
 
+#if defined(linux_HOST_OS)
+  Cardano.BM.Scribe.Systemd.plugin logConfig baseTrace switchBoard
```
Comment (CodiePP) [inline cardano-config/src/Cardano/Config/Logging.hs]: 'baseTrace' should be 'trace' so it can be turned on/off.

Comment (deepfire) [reply cardano-config/src/Cardano/Config/Logging.hs]: @CodiePP, done.

File: configuration/log-configuration.yaml [hunk 2]
```diff
@@ -76,6 +76,10 @@ options:
     cardano.node.peers.BlockFetchDecision:
       - kind: UserDefinedBK
         name: LiveViewBackend
+  mapScribes:
+    cardano:
+      - JournalSK
```
Comment (CodiePP) [inline configuration/log-configuration.yaml]: this should be added to 'defaultScribes'

Comment (deepfire) [reply configuration/log-configuration.yaml]: > this should be added to 'defaultScribes'

There is that old parsing problem popping up, which I mentioned previously:
```
defaultScribes:
  - - FileSK
    - "node-0.log"
  - - JournalSK
    - "foo"
```
..yields
```
AesonException "Error in $.defaultScribes[1][0]: The key \"JournalSK\" was not found"

cardano-node: Aeson exception:
Error in $.defaultScribes[1][0]: The key "JournalSK" was not found
```

**Comment by deepfire** [comment]
@CodiePP, waiting for your feedback on the design question re: plugin separation .

**Comment by CodiePP** [comment]
there was a way to integrate the lobemo-scribe-systemd plugin.
this PR is replaced with PR #502.

**Pull request closed without merging**